cpp/src/ray/api.cc

```diff
@@ -10,9 +10,11 @@ namespace api {
 std::shared_ptr<RayRuntime> Ray::runtime_ = nullptr;
 
 std::once_flag Ray::is_inited_;
-void Ray::Init() {
-  std::call_once(is_inited_,
-                 [] { runtime_ = AbstractRayRuntime::DoInit(RayConfig::GetInstance()); });
+void Ray::Init(std::string address, bool local_mode, int min_workers, int max_workers) {
```

Comment:
I prefer to make `Ray::Init()` having no parameters. You can get any config using `RayConfig::GetInstance()` in anywhere. And in fact in Java API, `Ray.init()` has no parameters too.